CHANGELOG.md

```diff
@@ -1,8 +1,35 @@
-CHANGELOG
-=========
+# Changelog
+All notable changes to this project will be documented in this file.
 
-## HEAD (Unreleased)
+## [Unreleased]
 _(none)_
 
----
+## [6.0.0 - Preview 3] - 2020-03-11
+The changelog below is a summary of the all of the preview releases.
 
+### Added
+- New gRPC .NET client added. 
+- The ability to filter reads by regex or prefix for both stream name and event type.
+- New options for certicates. Can be provided as PKCS, public/private key pair and windows store.
+- Introduction of a liveness health check at `{server_address}/health/live`.
+- Improvements to projection writes.
+- Read-only replica. A node that will not partake in elections and is non promotable. Started with the `--read-only-replica` argument
+- The ability to resign master using `{server_address}/admin/node/resign`.
+- Added in the ability to set node priority with `{server_address}/admin/node/priority/{priority}`.
+
+### Changed
```

Comment:
```suggestion
### Changed
```

Reply:
This will not render the markdown correctly is the suggested change is not applied.